pokemon_app/app/controllers/access_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class AccessController < ApplicationController
+  layout 'admin'
+
+  before_action :confirm_logged_in, except: %i[login attempt_login logout]
+
+  def menu
+    # @username = session[:username]
+  end
+
+  def login
+    # login form
+  end
+
+  def attempt_login
+    if params[:username].present? && params[:password].present?
+      found_user = User.where(username: params[:username]).first
+      authorized_user = found_user.authenticate(params[:password]) if found_user
+    end
+
+    if authorized_user
+      p '--------------auth'
+      session[:user_id] = authorized_user.id
+      session[:username] = authorized_user.username
+      flash[:notice] = 'You are now logged in.'
+      redirect_to(access_menu_path)
```

Comment:
TRy to avoid if possible the use of `()`. I would set this as `redirect_to access_menu_path`